packages/default-reporter/src/reporterForClient/reportContext.ts

```diff
@@ -0,0 +1,40 @@
+import { ContextLog, PackageImportMethodLog } from '@pnpm/core-loggers'
+import most = require('most')
+
+export default (
+  log$: {
+    context: most.Stream<ContextLog>,
+    packageImportMethod: most.Stream<PackageImportMethodLog>,
+  }
+) => {
+  return most.combine(
+      (context, packageImportMethod) => {
+        if (context.currentLockfileExists) {
+          return most.never()
+        }
+        let method!: string
+        switch (packageImportMethod.method) {
+          case 'copy':
+            method = 'copied'
+            break
+          case 'clone':
+            method = 'cloned'
+            break
+          case 'hardlink':
+            method = 'hard linked'
+            break
+          default:
+            method = packageImportMethod.method
+            break
+        }
+        return most.of({
+          msg: `\
+Packages are ${method} from the content-addressable store to the virtual store.
+\tContent-addressable store is at:\t${context.storeDir}
```

Comment:
A couple of comments:

- I don't think indents are necessary at all.
- Using tabs is a poor choice, different people use different sizes, so there is absolutely no guarantee the directory names will line up this way.
- 8 characters as an indent is just way too much. In the typographic world, that much is never used.
- If the goal is to line up the directories, just count the characters in the labels and add the requisite number of spaces after the colon.

Reply:
I used tabs because that is how Git indents similar output.
Although I like how it looks, I don't have objections using one space.

But there should be at least one space at the start. Otherwise, you can visually tell that those three lines are related.

Reply:
Two spaces will be much more visible.

Reply:
done